### PR TITLE
Sync Authorities Set

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: frame/bridge/ethereum/backing/src/lib.rs [hunk 1]
```diff
@@ -165,10 +172,11 @@ decl_storage! {
 	trait Store for Module<T: Trait> as DarwiniaEthereumBacking {
 		pub VerifiedProof
 			get(fn verified_proof)
-			: map hasher(blake2_128_concat) EthereumTransactionIndex => Option<bool>;
+			: map hasher(blake2_128_concat) EthereumTransactionIndex => bool = false;
```
Comment (HackFisher) [inline frame/bridge/ethereum/backing/src/lib.rs:175]: Looks good, just to double check again:

Removing Option will not affect storage.

I remember last time @AsceticBear also asked the same question.

Comment (AurevoirXavier) [reply frame/bridge/ethereum/backing/src/lib.rs:175]: I'm pretty sure. It won't affect the storage.

**Review by HackFisher — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Need some test data.

Maybe next PR, if @WoeOm are not ready.

File: frame/bridge/ethereum/backing/src/lib.rs (hunk 1 above)
Comment (AsceticBear) [reply frame/bridge/ethereum/backing/src/lib.rs:175]: It would be more convincing if more analysis about the storage influence provided or local test.

**Review by HackFisher — APPROVED** [review]

**Comment by WoeOm** [comment]
@wuminzhe

**Pull request merged**